packages/lambdas/src/ihe-inbound-document-query.ts

```diff
@@ -18,22 +19,19 @@ export const handler = Sentry.AWSLambda.wrapHandler(async (event: string) => {
 
   const result = await processInboundDocumentQuery(baseRequest.data, apiUrl);
 
-  if (
-    result.extrinsicObjectXmls &&
-    result.extrinsicObjectXmls.length > 0 &&
-    result.cxId &&
-    postHogSecretName
-  ) {
+  if (result.extrinsicObjectXmls && result.extrinsicObjectXmls.length > 0 && postHogSecretName) {
```

Comment:
This should always be greater than 0 given the way we are generating cdas on the fly right? 

Do we want to have > 1 here for analytics so it actually tells us usable info?